### PR TITLE
add delta sync enabled flag in replicator config

### PR DESCRIPTION
**Pull request by Sandychuang8**
update PR

### CODE REVIEW

File: src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs [hunk 1]
```diff
@@ -193,6 +193,17 @@ public ReplicatorType ReplicatorType
         [NotNull]
         public IEndpoint Target { get; }
 
+        /// <summary>
+        /// A bool value indicating the delta sync is enabled or not. The
+        /// default value it true.
+        /// </summary>
+        [NotNull]
```
Comment (borrrden) [inline src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs]: Remove this. [NotNull] [CanBeNull] is only for reference types

File: src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs [hunk 2]
```diff
@@ -193,6 +193,17 @@ public ReplicatorType ReplicatorType
...
+        /// <summary>
+        /// A bool value indicating the delta sync is enabled or not. The
+        /// default value it true.
+        /// </summary>
+        [CanBeNull]
+        public bool DeltaSyncEnabled
+        {
+            get => !Options.NoDeltas;
+            set => _freezer.PerformAction(() => Options.NoDeltas = !value);
+        }
```
Comment (Sandychuang8) [inline src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs]: @borrrden yes, DeltaSyncEnabled default value should be true. Not NoDelta is Delta.. On LiteCore, Jens uses "noDelta" as flag, but Pasin uses DeltaSyncEnabled on Lite.

Comment (borrrden) [reply src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs]: Oh, I see!  Things are flipped around here.

**Comment by borrrden** [comment]
Close this if it is no longer relevant.

**Comment by Sandychuang8** [comment]
Feature removed.

**Pull request closed without merging**